libs/application-generic/src/services/resource-validator.service.ts

```diff
@@ -0,0 +1,93 @@
+import { Injectable } from '@nestjs/common';
```

Comment:
This code applies only to the API so let's add it there. Reminder, applicarion-generic should be leaner moving forward.

Reply:
correct but today we have create-workflow at application generic, which is blocking from adding this service to API app.